ls1intum/Artemis
# [Exam] several style fixes & tooltips

### Checklist

### Motivation and Context


The save & continue / save button did not make sense and hiding it by `ngIf` caused problems with the layout of the nav.
On certain screen sizes the buttons in the exam navigation are overlapping. See https://github.com/ls1intum/Artemis/pull/1767#issuecomment-651818691

### Description

The save & continue / save button is now set to not visible in order to keep the layout.
The indicator of the exercise now switches earlier to the smaller display to prevent overlapping. (No overlap is only guaranteed for screen at least as wide as iPads (768px) but it also works until approx. 500px)
Added tooltips for the buttons.

### Steps for Testing


1. Log in to Artemis
2. Prepare an exam with at least one programming exercise
4. Check that the save & continue / save button is not shown when working on the programming exercise
5. Make sure the navigation is displayed correctly even when changing the window size
